pkg/boot/params.go

```diff
@@ -111,12 +115,17 @@ func MakeHypervisorParams(gwIP net.IP, sk cipher.SecKey) (Params, error) {
 		LocalPK:   pk,
 		LocalSK:   sk,
 	}
+	if wifiName != "" || wifiPass != "" {
```

Comment:
Why not just `if wifiName != `? Surely there is no point in setting these if only a password was received?